plop-templates/@react-types/src/index.d.ts.hbs

```diff
@@ -13,9 +13,15 @@
 import {DOMProps, StyleProps} from '@react-types/shared';
 
 export interface {{componentName}}Props {
-  
+  value?: any,
+  defaultValue?: any,
+  onChange?: any
 }
 
-export interface Spectrum{{componentName}}Props extends {{componentName}}Props, DOMProps, StyleProps {
+export interface Aria{{componentName}}Props extends {{componentName}}Props, DOMProps {
```

Comment:
any chance of conditionally including this export based on if they are creating an aria package?

Reply:
Yep, looking into it. I'll probably see if I can do a similar conditional for imports in the stately and aria hooks and perhaps the spectrum component as well

Reply:
👍🏻  there should be a big object at the end of doing all the prompts with all the answers